app/themes/user-admin/e2e/user-edit/user-edit.e2e-spec.ts

```diff
@@ -0,0 +1,110 @@
+import { UserAdminPage } from './user-edit.po';
+import { protractor, browser, element, by, WebElement } from 'protractor';
+
+describe('User Edit Page', function() {
+    let page: UserAdminPage;
+
+    beforeEach(() => {
+        page = new UserAdminPage();
+    });
+
+    it('should login successfully with right email and password', () => {
+        page.navigateTo();
+        let email = browser.findElement(protractor.By.name('email'));
+        let password = browser.findElement(protractor.By.name('password'));
+        let submit = browser.findElement(protractor.By.className('btn btn-success'));
+
+        email.sendKeys('foo@mail.com');
+        password.sendKeys('asdf1234');
+        
+        submit.click();
+
+        expect(page.getParagraphText("app-home h1")).toEqual('Welcome to the Erdiko User Admin');
+    });
+
+    it('should lead to User List page at click of the User List link', () => {
+        //Click UserList in Nav to get to UserList Page
+        let userListNav = browser.findElement(protractor.by.css('nav ul > li:nth-child(2) > a'));
```

Comment:
Targeting these elements might be a little easier if we use classes or IDs for some of these elements. Can you look into using more 'human readable' CSS selectors?